zebrad/build.rs

```diff
@@ -0,0 +1,12 @@
+extern crate vergen;
+
+use vergen::{generate_cargo_keys, ConstantsFlags};
+
+fn main() {
+    // Setup the flags, toggling off the 'SEMVER_FROM_CARGO_PKG' flag
+    let mut flags = ConstantsFlags::empty();
+    flags.toggle(ConstantsFlags::SHA_SHORT);
+
```

Comment:
Make sure that every build gets the latest commit hash
```suggestion
    flags.toggle(ConstantsFlags::REBUILD_ON_HEAD_CHANGE);

```